### ISSUE TITLE
filter by dropdown menue instead of free text

### ISSUE THREAD

**Issue by youneschaoui** [akveo/ng2-smart-table#175]
is it possible to use column filters whit dropdown predefined list of possible values (eg. sexe )

**Comment by Bramskyyy** [comment]
Yes this is possible. In your column config do something like this:

`
filter: {
	type: 'list',
		config: {
			selectText: 'Show all',
				list: [
					{ value: 'm', title: 'male' },
					{ value: 'f', title: 'female' },
				]
			}
		}
`

**Comment by ImenAbbes** [comment]
is it possible to use dropdown with predefined list of  values in the add row ??

**Comment by madhuinwiter** [comment]
@ImenAbbes Yes you can use. **Check Gender Part** in the below columns settings.

**filter:** type - list :- It means you can search by selecting one option in the dropdown
**editor:** type - list :- It means you can assign a value by selecting one option in the dropdown

`columns: {
      first_name: {
        title: 'First Name'
      },
      last_name: {
        title: 'Last Name'
      },
      email: {
        title: 'Email'
      },
      gender: {
        title: 'Gender',
        filter: {
          type: 'list',
          config: {
            selectText: 'Search By Gender',
            list: [{ value: 'male', title: Male' },{ value: 'female', title: 'Female' }],
          },
        },
        editor: {
          type: 'list',
          config: {
            list: [{ value: 'male', title: Male' },{ value: 'female', title: 'Female' }],
          },
        },
      }
    }`
